content/en/docs/installation/guide/operator-k3s.md

```diff
@@ -0,0 +1,297 @@
+---
+title: Install Armory in Lightweight Kubernetes (K3s) using the Armory Operator
+linkTitle: Install in AWS EC2 using Operator
+weight: 50
+description: >
+  For POCs: Use Armory Operator to install Armory in a K3s instance running on an AWS EC2 VM
+---
+
+## Overview
+
+This guide walks you through using the [Armory Operator]({{< ref "operator" >}}) to install Armory in a [Lightweight Kubernetes (K3s)](https://k3s.io/) instance running on an AWS EC2 instance. The environment is for POCs and development only. It is **not** meant for production environments.
+
+See the [Install on Kubernetes]({{< ref "install-on-k8s" >}}) guide for how to install Spinnaker using the Armory Operator in a regular Kubernetes installation.
```

Comment:
```suggestion
See the [Install on Kubernetes]({{< ref "install-on-k8s" >}}) guide for how to install Spinnaker™ using the Armory Operator in a regular Kubernetes installation.

Reply:
First mention of Spinnaker on a page needs the ™ symbol. It's the other half of the renaming project.

Reply:
what a PITA. Should we just create a shortcode for Spinnaker?  something like:

spinnaker.html
```
Spinnaker<sup>TM</sup>
```

```
install open source {{< spinnaker >}}
```

Reply:
The keyboard shortcut for ™ is `alt + 2`